Reservia_files/style.css

```diff
@@ -0,0 +1,620 @@
+
```

Comment:
didn't we say this file should be splet in many smallers one ? to improve readability and maintenance?

Reply:
Indeed, we said that this file should be broken down into several small parts. Here is the new file. Cordially.

Reply:
I proceeded to the factorization of the code, according to the repetition of the code.